src/storm-pomdp-cli/storm-pomdp.cpp

```diff
@@ -358,6 +358,7 @@ namespace storm {
                     std::string transformMode = transformSettings.getFscApplicationTypeString();
                     auto pmc = toPMCTransformer.transform(storm::transformer::parsePomdpFscApplicationMode(transformMode));
                     STORM_PRINT_AND_LOG(" done.\n");
+
```

Comment:
Please undo this change as this file should not be affected by an extension to parameter lifting